app/main.py

```diff
@@ -1,8 +1,18 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result_list = [Person(person["name"], person["age"]) for person in people]
+
+    for num, person in enumerate(people):
```

Comment:
In this case, you do not need to use an enumerator and access the list by index, it is better to iterate through the list and address a specific object (person).